commands/viewport.go

```diff
@@ -48,6 +48,19 @@ func (cmd *Viewport) Parse() error {
 		cmd.scrollFullPage(1)
 	case "pgup", "pageup":
 		cmd.scrollFullPage(-1)
+	case "high":
+		_, ymax := songlistWidget.GetVisibleBoundaries()
```

Comment:
The code is identical across the three cases, except the delta parameter. Can you put this in a separate function?

Reply:
I wouldn't say identical, but sure, similar. I've resolved this in the latest iteration -- please take another look.